### PR TITLE
BPS-842 Add  smoke tests config

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-842

### Change description ###
Add smoke tests config:
- copy payments queue secret from vault 
- set pay hub url 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -10,13 +13,51 @@ def component = "payment-processor"
...
+      def serviceBusSecret = "servicebus-secret-namespace-${aksServiceName}"
+
+      def namespace = new TeamConfig(this).getNameSpace(product)
+      def kubectl = new Kubectl(this, subscription, namespace)
+      kubectl.login()
+
+      // Get payments queue connection string
+      def sbConnectionStr = kubectl.getSecret(serviceBusSecret, namespace, "{.data.connectionString}")
+      env.PAYMENTS_QUEUE_READ_CONNECTION_STRING = "${sbConnectionStr};EntityPath=payments"
+      env.PAY_HUB_URL = "http://ccpay-bulkscanning-api-aat.service.core-compute-aat.internal"
```
Comment (nitinprabhu) [inline Jenkinsfile_CNP]: do you need this ?

Comment (aliveni) [reply Jenkinsfile_CNP]: The pay hub url? Yes

Comment (nitinprabhu) [reply Jenkinsfile_CNP]: are you not setting it via `values.preview.template.yaml`? and this env var is used by smoke/functional or something else ?

Comment (aliveni) [reply Jenkinsfile_CNP]: As discussed, will remove the env vars here

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -10,13 +13,51 @@ def component = "payment-processor"
 def channel = '#bsp-build-notices'
 GradleBuilder builder = new GradleBuilder(this, product)
 
+def secrets = [
+  'bulk-scan-${env}': [
+    secret('payments-queue-listen-connection-string', 'PAYMENTS_QUEUE_READ_CONNECTION_STRING')
+  ]
```
Comment (mdayican) [inline Jenkinsfile_CNP]: do you need this?
inline 48 you get this again?

Comment (aliveni) [reply Jenkinsfile_CNP]: As nitin suggested earlier, I'll remove reading the secret in line 48. so we'll need this

File: src/smokeTest/resources/application.conf [hunk 3]
```diff
@@ -1,3 +1,4 @@
 test-url = "http://localhost:8583"
 test-url = ${?TEST_URL}
-
+test-pay-hub-url = ${TEST_PAY_HUB_URL}
+payments-queue-connection-string = ${PAYMENTS_QUEUE_READ_CONNECTION_STRING}
```
Comment (rkondratowicz) [inline src/smokeTest/resources/application.conf]: would be best if `payments-queue-connection-string` contained 'read' imho.

Comment (aliveni) [reply src/smokeTest/resources/application.conf]: Renamed

**Pull request merged**